app/controllers/posts_controller.rb

```diff
@@ -9,7 +9,37 @@ def index
   def show
     @user = User.find(params[:user_id])
     @post = @user.posts.find(params[:id])
+    @comments = @post.comments
+    @comment = Comment.new
 
     render :show
   end
+
+  def new
+    @user = current_user
+    @post = Post.new
+  end
+
+  def create
+    # title = post_params[:title]
+    # text = post_params[:text]
+    # posts = Post.new(author_id: current_user.id, title: title, text: text, comments_counter: 0, likes_counter: 0 )
+    @post = current_user.posts.build(post_params)
+    @post.comments_counter = 0
+    @post.likes_counter = 0
+
+    if @post.save
+      # redirect_to user_posts_path(current_user, @post), notice: 'Post created successfully'
```

Comment:
- [ ] Please do well to remove commented code from your project. This keeps your codebase clustered is done repeatedly wich won't be a good practice